pkg/controller/jenkins/configuration/base/reconcile.go

```diff
@@ -37,6 +42,10 @@ const (
 	fetchAllPlugins = 1
 )
 
+var (
+	routeAPIFound = false
```

Comment:
better to externalize this in another file of package as it is not related specifically to the reconciler